packages/react-storage/src/components/StorageBrowser/views/Controls/Table.tsx

```diff
@@ -155,6 +165,26 @@ const LocationDetailViewColumnSortMap = {
   size: compareNumbers,
 };
 
+const LocationDetailViewColumnEmptyHeaderMap = ['download'];
+
+export const LocationDetailSelectAllControl = ({
```

Comment:
Think we can just use the `Checkbox` directly at the call site of `LocationDetailSelectAllControl` in the `LocationDetailView` and remove this abstraction layer

Reply:
updated